user/stubs.c

```diff
@@ -0,0 +1,119 @@
+#include "userargs.h"
+#include <sys/syscall.h>
+
+sgxlkl_userargs_t* __sgxlkl_userargs;
+
+void sgxlkl_warn(const char* fmt, ...);
+
+int snprintf(char *str, size_t size, const char *format, ...);
+
+/*
+**==============================================================================
+**
+** syscall:
+**
+**==============================================================================
+*/
+
+long lkl_syscall(long no, long* params)
```

Comment:
Does this need a hidden-visibility attribute, or is the build system still hiding this from userspace?